src/main/resources/templates/download.html

```diff
@@ -2,7 +2,7 @@
 
 <html xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
 <head th:include="fragments/head.html::head"></head>
-<body>
+<body th:class="${isGovukBranded ? 'gov-uk' : 'openregister-org'}">
```

Comment:
the javabean-convention magic means that you would be able to just say `brandingClassName` in the template and it will get rewritten to `getBrandingClassName()`